ush/load_fv3gfs_modules.sh

```diff
@@ -34,7 +34,7 @@ elif [[ -L /usrx && "$( readlink /usrx 2> /dev/null )" =~ dell ]] ; then
 	module load module_base.wcoss_dell_p3 
 elif [[ -d /lfs/h2 ]]; then
     # We are on WCOSS2 (Cactus or Dogwood)
-	module load module_base.wcoss2
+	module load module_base.wcoss2.lua
```

Comment:
@GuangPingLou-NOAA Please remove the `.lua` part on line 37 and I will submit this PR. I'll convert the module file to LUA when I convert the other modulefiles and then resolve any issues related to that. Thanks!